Dockerfile

```diff
@@ -251,7 +237,12 @@ RUN cargo install --version=${wizer_version} wizer --all-features
 
 # Install crosvm.
 # We're not interested in most of the features in crosvm (e.g. wayland support), but GDB support would be nice.
-RUN cargo install --git https://chromium.googlesource.com/chromiumos/platform/crosvm/ --rev 31f04e92709980a4ffc56b1631f8b4be437cc2fe crosvm --no-default-features --features gdb
+RUN cargo install \
+  --git=https://chromium.googlesource.com/chromiumos/platform/crosvm/ \
```

Comment:
While you're in here cleaning up things, crosvm has updated its repository to https://chromium.googlesource.com/crosvm/crosvm; consider pulling in a newer revision from there.

Reply:
I tried naively but it didn't seem to work, created #3332 to track.

Reply:
Yeah, I don't know if the commit hashes survived the transition. You could just pick the latest revision from that new repository.

Reply:
I did try with the latest commit from that revision indeed, but that also didn't work.